### PR TITLE
Clarify that the Platform API on Drupal 8

### PR DESCRIPTION
**Pull request by greg-1-anderson**
## Effect
The existing text caused some confusion for a user who thought that no API module being available for Drupal 8 meant that there was some missing functionality for this version of Drupal. Clarified that the API module is not available because it is not necessary, and that this module is not intended to be used as an API by end users (and therefore probably should not have been named "API").

## Remaining Work
- [x] Copy Review

## Post Launch
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5492 | Difference |
| --- | --- | --- | --- |
| Performance Score | `81` | `41` | `-40` |
| Best Practice Score | `93` | `86` | `-7` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `96` | `-2` |

Lighthouse Audit Reports:

- [`pr-5492` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18180/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18180/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: source/content/modules.md [hunk 1]
```diff
@@ -34,9 +34,9 @@ This module provides general methods your site needs to access aspects of the in
 The code for the API module is available within our upstream repositories, in the `modules/pantheon` directory. The functionality of this module is provided in three parts: Apache Solr, Pantheon Platform API, and Login.
 
 ### [Pantheon Platform API](https://github.com/pantheon-systems/drops-7/blob/master/modules/pantheon/pantheon_api/pantheon_api.info)
-This module provides general methods your site needs to access aspects of the internal Pantheon API. This is necessary for clearing caches and other common workflows on the platform.
+This module provides general methods your site needs to access aspects of the internal Pantheon API. This is necessary for clearing caches and other common workflows on the platform. Currently, there is no API module necessary for Drupal 8.
```
Comment (alexfornuto) [inline source/content/modules.md]: @greg-1-anderson should the previous sentence to your addition be modified to specify D7? As is, the two seem slightly contradictory.

Comment (greg-1-anderson) [reply source/content/modules.md]: Ah, yeah. The path says "drops-7", but that is too subtle. This module is for Drupal 7 only.

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5492 | Difference |
| --- | --- | --- | --- |
| Performance Score | `59` | `52` | `-7` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5492` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18253/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/18253/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Pull request merged**